src/pages/api/v3/pull.ts

```diff
@@ -57,66 +57,35 @@ export default async function handler(
         });
         //#endregion  //*======== Create Client Group if not existent ===========
 
-        if (!cookie) {
```

Comment:
This branch identical to other one, modulo defaulting cookie to zero.

in the case that the cookie incoming cookie is > 0, but the corresponding client record is not found, should really throw an error. Because in that case the server has lost state and sync cannot continue.